regression/cbmc-java/destructor1/test.desc

```diff
@@ -1,6 +1,6 @@
 CORE
 Break.class
---show-goto-functions
+--show-goto-functions --no-core-models
```

Comment:
Add a comment to the .desc file, after a second `--` delimiter, e.g.
```
musthave1
musthave2
--
mustnothave1
mustnothave2
--
comments here
```
Explain in the comment why this test has --no-core-models